stdlib/3.9/zoneinfo/__init__.pyi

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import os
+import typing
+from datetime import datetime, tzinfo
+from typing import IO, Optional, Sequence, Type, Union
+
+class ZoneInfo(tzinfo):
+    _T = typing.TypeVar("_T", bound="ZoneInfo")
+    key: str
+    def __init__(self, key: str) -> None: ...
+    @classmethod
+    def no_cache(cls: Type[_T], key: str) -> _T: ...
+    @classmethod
+    def from_file(
+        cls: Type[_T], fobj: IO[bytes], /, key: Optional[str] = None
```

Comment:
```suggestion
        cls: Type[_T], __fobj: IO[bytes], key: Optional[str] = ...
```

The `/` syntax for positional-only arguments is not supported in stubs, because type checkers using typed-ast (e.g. mypy when run with Python < 3.8) will not be able to parse this file. See also the discussion in python/typed_ast#118. The double underscore is a workaround understood by type checkers.

Default values must be `...`.